addon/mixins/in-viewport.js

```diff
@@ -208,13 +213,13 @@ export default Mixin.create({
         window.cancelAnimationFrame(rAFIDS[elementId]);
         delete rAFIDS[elementId];
       });
+    } else {
```

Comment:
The `viewportListeners` only get setup `if (!get(this, 'viewportUseRAF'))` so we should unbind only if they got setup.